### ISSUE TITLE
Feral Spirit [Shaman Talent] - Wrong level for summoned Spirit Wolves.

### ISSUE THREAD

**Issue by arielcami** [azerothcore/azerothcore-wotlk#12048]
### Current Behaviour

When the Shaman talent Feral Spirit is used, the wolves that spawn, are level 13.

https://user-images.githubusercontent.com/71854140/173381029-e4e0c3f8-1f8a-4f5b-955f-b69ce7d0dd0d.mp4

### Expected Blizzlike Behaviour

Wolves should be the same level that the master is.

### Source

Look at the summoned wolves level at 0:12

https://www.youtube.com/watch?v=rQvRYjFfoCA

### Steps to reproduce the problem

1. Get a Shaman and level it up to any level that Feral spirit talent is unlocked.
2. Learn Feral spirit.
3. Use Feral spirit.

### AC rev. hash/commit

AC rev hash/commit
d236dc9d078f+

### Operating system

Win10 x64

### Custom changes or Modules

mod-eluna

**Comment by temperrr** [comment]
Mine are level 60 when I am level 60
![image](https://user-images.githubusercontent.com/20357406/173388709-c4adda41-d275-4c26-878b-0c2b020a7b87.png)

**Comment by arielcami** [comment]
Can you show your commit please?

**Comment by temperrr** [comment]
![image](https://user-images.githubusercontent.com/20357406/173405234-076a04d9-c620-426c-b8f2-a807d8ab725d.png)
Even after respeccing etc it shows 63 so indeed a problem

**Comment by temperrr** [comment]
hmm when I tried locally it does work fine :?

**Comment by temperrr** [comment]
![image](https://user-images.githubusercontent.com/20357406/173506611-d83f2290-46c8-4f2d-a4f4-d30c87a868c5.png)

**Comment by arielcami** [comment]
I discover what was the problem...

In worldserver.conf > MaxPlayerLevel was lower that 80.

So when this is set to 60 for instance, Wolves can only be 3 levels ahead of the master. 
That's why your wolves were 63.

In my server I set it to lvl 10, so wolves were 13.

The bug happens when the level cap is server locked and you override that with .level command.

![error](https://user-images.githubusercontent.com/71854140/173649807-3487c152-7718-4705-9a33-f80456e4a21b.png)
Fixed.
